lib/SILOptimizer/Mandatory/SendNonSendable.cpp

```diff
@@ -0,0 +1,33 @@
+#include "../../Sema/TypeCheckConcurrency.h"
```

Comment:
ehh I spotted `#include "../AST/InlinableText.h"` in `TypeCheckMacros` so maybe it's OK. will defer to @DougGregor on this.

Reply:
I probably shouldn't have done it there, either. I agree that we should move that function out to a separate header. Again, it can be a follow-up PR.